capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
+end
```

Comment:
- [ ] Please use the `capitalize` method instead of upcase, Upcase will capitalize only the first word of the string.
- You can consider the below code: 
```ruby
def correct_name
      @nameable.correct_name.capitalize
    end

Reply:
Thank you for the feedback. I changed it from `upcase` to `capitalize`. However, I want to mention that there might be a mistake in what you said! `upcase` won't just capitalize the first word in a string, it will actually capitalize the whole string. On the other hand, `capitalize` will capitalize only the first letter of the string and converts the rest of the letters to lowercase.